readme.md

```diff
@@ -68,495 +35,38 @@ From the npm registry:
 > yarn/pnpm add express-rate-limit
 ```
 
-From Github Releases:
-
-```sh
-# Using npm
-> npm install https://github.com/express-rate-limit/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
-# Using yarn or pnpm
-> yarn/pnpm add https://github.com/express-rate-limit/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
-```
-
-Replace `{version}` with the version of the package that you want to your, e.g.:
-`6.0.0`.
-
-## Usage
-
-### Importing
-
-This library is provided in ESM as well as CJS forms, and works with both
-Javascript and Typescript projects.
-
-**This package requires you to use Node 16 or above.**
-
-Import it in a CommonJS project (`type: commonjs` or no `type` field in
-`package.json`) as follows:
-
-```ts
-const { rateLimit } = require('express-rate-limit')
-```
-
-Import it in a ESM project (`type: module` in `package.json`) as follows:
-
-```ts
-import { rateLimit } from 'express-rate-limit'
-```
-
-### Examples
+> Note that this library requires you to use `node` version 16 or above.
 
-To use it in an API-only server where the rate-limiter should be applied to all
-requests:
+An example of its use is as follows:
 
 ```ts
+// const { rateLimit } = require('express-rate-limit') // Use this for `commonjs` projects.
 import { rateLimit } from 'express-rate-limit'
 
 const limiter = rateLimit({
 	windowMs: 15 * 60 * 1000, // 15 minutes
-	limit: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
+	limit: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes).
 	standardHeaders: 'draft-7', // draft-6: `RateLimit-*` headers; draft-7: combined `RateLimit` header
-	legacyHeaders: false, // Disable the `X-RateLimit-*` headers
-	// store: ... , // Use an external store for more precise rate limiting
+	legacyHeaders: false, // Disable the `X-RateLimit-*` headers.
+	// store: ... , // Use an external store for more precise rate limiting.
```

Comment:
The default store is fairy precise now, external stores are more for consistency across servers, processes, and reboots

```suggestion
	// store: ... , // Use an external store for consistency across instances.
	// Additional configuration options - see https://express-rate-limit.mintlify.app/refference/configuration
```